### DISCUSSION TITLE
No longer asks me where to save

### DISCUSSION THREAD

**Discussion by BarbaraG526** [aclap-dev/video-downloadhelper#348] [Help and Q&A]
### Your question

The app no longer asks me where to save the download after I click "Save As".  This is a huge inconvenience because now I have to go find the download and move it to the folder I want it in.  This is a recent change, and I have been waiting for a fix because I assumed it was a glitch of some kind.  I have always loved this app, but now it is costing me time and effort, so I'm really hoping this can revert somehow to allowing me to select the folder after I click "Save As".  Can someone please address this and help me to correct it?  

### Addon & CoApp version and details

![image](https://github.com/aclap-dev/video-downloadhelper/assets/161066946/14962e7a-f54c-48c7-a759-0be5e23d0cc1)

**Comment by Mattmjs** [comment]
Where are you seeing this happen, do you have an example.

**Comment by BarbaraG526** [comment]
I click on the icon in my tool bar, select the video and click the arrow to the right, select Download and Convert.  the window comes up with the file type I have previously selected, and the "Save As" button.

In the past, clicking on the save as button would open a page where I could see the folders on my computer or the last folder that I saved to.  Now, it just starts downloading the video and doesn't ask me where to save it.  It saves it to the last folder I saved to.  I even tried to change that in settings by selecting Behavior, but for the download option it only offers a default folder.  There is no way in settings to have it ask me where to save it.  Does that help?

**Reply by Mattmjs** [reply]
Ok I don't use download and convert myself, but I tried it in Linux and it only gives a dialog to choose the browser or companion app
for the download. It didn't ask where to save it. I chose it to convert to MOV but it remained as mp4.
So I don't know if conversion is broken. Does your conversion work ? Might need @paulrouget  to look at this.
I'm just another user.

**Reply by BarbaraG526** [reply]
Thank you for your reply.  Honestly, I understand so little about this.  I think my conversion is working.  I have just used the one mode in the menu, and it has always worked.  Then some changes were made to the app a while back, then I was prompted to get the Companion app (if I remember correctly), which I did.  I can't remember much about what I have changed, what has changed with the app or what the Conversion app even is.  I just want it to work for me the way it used to.  I need it to ask me every time where to save the download.  I don't want it saved to a default folder or to the last folder it saved to.  I really appreciate your trying to help me out on this. So, yes, it would be helpful if @paulrouget could assist with this.  Thank you so much!

**Reply by BarbaraG526** [reply]
Okay, I just tried it with just the Download selection, not Download and Convert. After clicking Save As, it took me to the page to select where I want to save it. This is what I want to have happen. Thank you @Mattmjs for asking me that question.I was downloading an mp4 and saving as an mp4, so no conversion needed.

 I will continue with this process until a conversion is required. I would like to understand if a conversion is required, and I choose Download and Convert, how to get it to take me to the same page to select my folder.

**Comment by ericnyc1** [comment]
I have noticed the same issue after updating the driver.
